src/core/operation.h

```diff
@@ -27,7 +27,7 @@ typedef enum QUIC_OPERATION_TYPE {
     QUIC_OPER_TYPE_UNREACHABLE,         // Process UDP unreachable event.
     QUIC_OPER_TYPE_FLUSH_STREAM_RECV,   // Indicate a stream data to the app.
     QUIC_OPER_TYPE_FLUSH_SEND,          // Frame packets and send them.
-    QUIC_OPER_TYPE_TLS_COMPLETE,        // A TLS process call completed.
+    QUIC_OPER_TYPE_DEPRECATED,          // No longer used.
```

Comment:
Since this isn't a public file, can't we just remove this?

Reply:
Nevermind, it would break all logs, wouldn't it.

Reply:
The problem is that these map to the ETW manifest, which is semi-public. At the very least, our Windows builds will have them for some time.